app/src/main/java/org/mozilla/focus/widget/FirefoxProgressBar.kt

```diff
@@ -35,9 +35,11 @@ class FirefoxProgressBar @JvmOverloads constructor(
         browserFrag.session.loading.observe(browserFrag, object : NonNullObserver<Boolean>() {
             public override fun onValueChanged(loading: Boolean) {
                 if (loading) {
```

Comment:
Another bug I noticed, we need to cancel any pending animation messages here as well.